### PR TITLE
Event head and trigger v3

### PR DESCRIPTION
**Pull request by MaxNoe**
These two new versions were also in the file provided by @FrancaCassol

### CODE REVIEW

File: eventio/simtel/objects.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ''' Implementations of the simtel_array EventIO object types '''
 import numpy as np
-from io import BytesIO
+from io import BytesIO, SEEK_END
```
Comment (dneise) [inline eventio/simtel/objects.py]: I'm on the phone, and somehow do not see where SEEK_END is needed....

File: tests/simtel/test_simtelfile.py [hunk 2]
```diff
@@ -182,3 +182,11 @@ def test_skip_calibration_events():
...
     with SimTelFile(frankenstein_path) as f:
         assert len(f.telescope_descriptions) == f.n_telescopes
+
+
+def test_new_prod4():
+    with SimTelFile('tests/resources/prod4_pixelsettings_v3.gz') as f:
+        i = 0
+        for e in f:
+            i += 1
+        assert i == 10
```
Comment (dneise) [inline tests/simtel/test_simtelfile.py]: Can we also test here for the new features?
 - plane wavefront comp
 - the 2 times

Comment (MaxNoe) [reply tests/simtel/test_simtelfile.py]: done

File: eventio/simtel/objects.py (hunk 1 above)
Comment (MaxNoe) [reply eventio/simtel/objects.py]: removed it

**Pull request merged**